scripts/hudson/quickstart.sh

```diff
@@ -82,6 +83,22 @@ function get_bt_dependencies {
   fi
 }
 
+# ensure the microprofile-lra artifact is in the local maven repository
+function build_microprofile-lra {
```

Comment:
isn't dangerous to use here `build_microprofile-lra` when it was recently changed in `narayana.sh` for mac to just underscores?